src/Misc.ino

```diff
@@ -1561,19 +1561,37 @@ String parseTemplate(String &tmpString, byte lineSize)
                           value = toString(UserVar[y * VARS_PER_TASK + z], ExtraTaskSettings.TaskDeviceValueDecimals[z]);
 
                         int oidx;
-                        if ((oidx = valueFormat.indexOf('O')) >= 0) // Output
+                        if (value.toFloat() == 0 || value.toFloat() == 1) //if val != 0 or != 1 ignore the format setting
```

Comment:
regarding toFloat(), you are right. Will change that.
The reason why I did the check on 0 or 1 only is because the #O format is valid only if value is 1 or 0, so for GPIO, not for any float value.

Reply:
The toFloat() operation was already in the original code.
And it was already doing the casting.
What then if i use toInt() instead of toFloat()?